x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_profiles/get_current.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { loginUsers, suggestUserProfiles } from '../../../../common/lib/user_profiles';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { superUser } from '../../../../common/lib/authentication/users';
+import { createCase, deleteAllCaseItems } from '../../../../common/lib/utils';
+import { getPostCaseRequest } from '../../../../common/lib/mock';
+import { getUserInfo } from '../../../../common/lib/authentication';
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getService }: FtrProviderContext) {
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const es = getService('es');
+  const security = getService('security');
+
+  describe('user_profiles', () => {
+    describe('get_current', () => {
```

Comment:
I think we should add some tests to test the `updated_by` and `closed_by` attributes in all of our SO (if we are not doing this already).